std/typecons.d

```diff
@@ -797,6 +845,29 @@ template Tuple(Specs...)
             import std.conv : to;
             return this.to!string;
         }
+
+        ///
+        unittest
```

Comment:
I'm pretty sure unit tests inside templated types have issues with one test being generated for each template instantiation; while it's tempting to use a documented unit test, it should be moved outside, and an `Example:` DDoc section used instead. The body of the unit test can be duplicated in both locations if the same effect is desired.